cupy-feedstock/recipe/build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+export NCCL_LIB_DIR="${PREFIX}/lib"
+export NCCL_INCLUDE_DIR="${PREFIX}/include"
+
+# To find libcuda.so.  Not sure why this is required...
+export LDFLAGS=-L/usr/lib64
```

Comment:
It is better to extend the existing LDFLAGS using `export LDFLAGS="-L/usr/lib ${LDFLAGS}"`.